modules/account-lib/src/coin/hbar/utils.ts

```diff
@@ -133,6 +132,22 @@ export function isValidAmount(amount: string): boolean {
   );
 }
 
+/**
+ * Returns whether the provided raw transaction has a valid format and type
+ *
+ * @param {any} rawTransaction - The raw transaction to be checked
+ * @returns {boolean} the validation result
+ */
+export function isValidRawTransactionFormat(rawTransaction: any): boolean {
```

Comment:
this is a good check. it might be good to indicate this is not the native raw format, but instead bitgo's preferred format